ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusMessageReply.qml

```diff
@@ -106,8 +106,8 @@ Item {
                 }
 
                 Item {
-                    implicitWidth: messageContentsLayout.implicitWidth
-                    implicitHeight: messageContentsLayout.implicitHeight
+                    Layout.fillWidth: true
+                    Layout.preferredHeight: messageContentsLayout.implicitHeight
```

Comment:
This is the actual fix ;) We are inside a Layout so the `Item` was missing propeer width (and height)